tools/platformio-build-esp32s2.py

```diff
@@ -262,7 +262,7 @@
         "UNITY_INCLUDE_CONFIG_H",
         "WITH_POSIX",
         "_GNU_SOURCE",
-        ("IDF_VER", '\\"v4.4-dev-479-g1067b2870-dirty\\"'),
+        ("IDF_VER", '\\"v4.4-dev-744-g1cb31e509\\"'),
```

Comment:
Hi @me-no-dev ! Could  you please clarify which commit of ESP-IDF is used? The commit message refers to `1cb31e509` and this macro points to `g1cb31e509`.

Reply:
This is the same. `g` means github commit `1cb31e509`